pod/perlhacktips.pod

```diff
@@ -362,9 +362,12 @@ It's best then to:
 
 =over
 
-=item B<Don't begin a symbol name with an underscore>; (I<e.g.>, don't
+=item B<Don't begin a file-level symbol name with an underscore>; (I<e.g.>, don't
 use: C<_FOOBAR>)
 
+It is fine to have a symbol in a function or block begin with an
+underscore.
+
```

Comment:
This implies you can use `_FOOBAR` at non-file-level scopes, but that is reserved at all scopes.